search/views/bizdev_search_core_users.view.lkml

```diff
@@ -15,9 +14,9 @@ view: bizdev_search_core_users {
               COALESCE(SUM(ad_click), 0) AS ad_click
          FROM
               search.search_clients_engines_sources_daily
-        -- WHERE
-        --     {% incrementcondition %} submission_date {%  endincrementcondition %}
-        --     -- AND submission_date >= DATE("2019-01-01")
+        WHERE
+            {% incrementcondition %} submission_date {%  endincrementcondition %}
+            AND submission_date >= DATE("2019-01-01")
```

Comment:
So that's the only way to specify start date for aggregation?

Reply:
Yup, this seems like the way to do it. Found it here: https://docs.looker.com/reference/view-params/increment_key#creating_an_incremental_sql-based_pdt